app/presenters/contents_list_presenter.rb

```diff
@@ -0,0 +1,9 @@
+class ContentsListPresenter
+  def initialize(contents_list)
+    @contents_list = contents_list
+  end
+
+  def to_component
+    @contents_list.items.map { |item| { href: item.href, text: item.text } }
```

Comment:
Why does this feel like it's a duplicate of something? Perhaps the ContentsList model should have a `to_h` method that returns this?

Reply:
(this file no longer present)